### ISSUE TITLE
Modify build_ogc_app.py to break process into independent steps that can be run independently

### ISSUE THREAD

**Issue by mcduffie** [unity-sds/unity-app-generator#13]
Modify build_ogs_app.py to have a command line interface similar to Git or Docker where subcommands are run from a directory  and can be independently called. This would better fit with the CI/CD process and make the program usable manually without requiring all steps be run.

**Comment by mcduffie** [comment]
Implemented at 1b9749216a4230e30f5109b87fffe22c6404f05d

**Comment by mcduffie** [comment]
Example usage:
```
tmp_dir=/tmp/app_gen_test

rm -rf $tmp_dir

build_ogc_app.py init \
    git@github.com:unity-sds/unity-example-application.git $tmp_dir

cd $tmp_dir
build_ogc_app.py build_docker
build_ogc_app.py push_docker quay-development-alb-656354332.us-west-2.elb.amazonaws.com:8000/mcduffie
build_ogc_app.py build_cwl
```
